apps/er-coap/er-coap-observe.c

```diff
@@ -58,9 +58,9 @@ LIST(observers_list);
 /*---------------------------------------------------------------------------*/
 /*- Internal API ------------------------------------------------------------*/
 /*---------------------------------------------------------------------------*/
-coap_observer_t *
+static coap_observer_t *
```

Comment:
I agree using `const` would be best but this seems pretty orthogonal to this PR, so I'd have it in another one. This PR is a nice cleanup and solved a known bug (https://github.com/contiki-os/contiki/pull/1250) so I'd focus on getting it in first.